src/widgets.js

```diff
@@ -264,7 +264,7 @@ var Widgets = {
      * the exports of a widget's module.
      */
     getDefaultAlignment: function(type, enabledFeatures) {
-        var widgetInfo = widgets[type];
+        var alignment, widgetInfo = widgets[type];
```

Comment:
To match https://github.com/Khan/style-guides , this should just be a variable declared on its own line, like:

```javascript
var alignment;
var widgetInfo = widgets[type];
```

(in addition to avoiding the weirdness of js single-line variable declarations, this has the benefit of not looking like destructuring)